codelists/migrations/0056_data_migration_add_coding_system_version.py

```diff
@@ -0,0 +1,32 @@
+# Generated by Django 4.1.1 on 2022-10-21 07:21
+
+from django.db import migrations
+
+
+def add_initial_coding_system_version(apps, schema_editor):
+    CodingSystemVersion = apps.get_model("versioning", "CodingSystemVersion")
+    CodelistVersion = apps.get_model("codelists", "CodelistVersion")
+    initial_versions = dict(
+        CodingSystemVersion.objects.filter(version="unknown").values_list(
+            "coding_system", "slug"
+        )
+    )
+    initial_versions.update({"opcs4": "none", "null": "none"})
```

Comment:
It could be a FK; I had dithered about that before.  It's "none" because it's non-nullable, which I think it should be for any new codelist versions we make.  But, there are old opcs4 ones that don't have any coding system models, so won't have a CodingSystemVersion that can be looked up.  This was a way to check for that.